web/web/auth/auth.py

```diff
@@ -9,5 +13,42 @@
 
 @auth_bp.route('/')
 @auth_bp.route('/auth', strict_slashes=False)
+@jwt_optional
 def index():
-    return render_template('auth/login.html')
+    access_token = request.cookies.get('access_token')
+    refresh_token = request.cookies.get('refresh_token')
+    if access_token and refresh_token:
```

Comment:
Why would we deny someone from accessing the app if the refresh token is expired or not present?

Reply:
Ah, I was thinking both were required to enter the app. Not sure why someone would have an access token without a refresh token, but can change to only access token required to be redirected

Reply:
I think a user should have both tokens (though it's not required), but to access the app, I would expect the user to send over only the access token.

Reply:
got it, thanks!